src/View/Cell.php

```diff
@@ -176,6 +176,17 @@ public function render($template = null)
         }
 
         $render = function () use ($template) {
+            try {
+                $reflect = new ReflectionMethod($this, $this->action);
+                $reflect->invokeArgs($this, $this->args);
```

Comment:
Moving this at the top of the function means the cell method will be always executed even if cached data always exists, only view rendering will be avoided. My intention when adding this feature was to avoid method invoking the method too as explained [here](https://github.com/cakephp/cakephp/pull/7455#discussion_r40501237)